experimental/traffic-portal/src/app/api/testing/profile.service.ts

```diff
@@ -220,4 +220,67 @@ export class ProfileService {
 		return this.profiles.splice(index, 1)[0];
 	}
 
+	private lastParamID = 20;
+	private readonly parameters:  ResponseParameter[] = [
+		{
+			configFile: "cfg.txt",
+			id: 1,
+			lastUpdated: new Date(),
+			name: "param1",
+			profiles: [],
+			secure: false,
+			value: "10"
+		}
+	];
+
+	public async getParameters(id: number): Promise<ResponseParameter>;
+	public async getParameters(): Promise<Array<ResponseParameter>>;
+	/**
+	 * Gets one or all Parameters from Traffic Ops
+	 *
+	 * @param id The integral, unique identifier (number) of a single parameter to be returned.
+	 * @returns The requested parameter(s).
+	 */
+	public async getParameters(id?: number): Promise<ResponseParameter | Array<ResponseParameter>> {
+		if (id !== undefined) {
+			const parameter = this.parameters.filter(t=>t.id === id)[0];
+			if (!parameter) {
+				throw new Error(`no such Parameter: ${id}`);
+			}
+			return parameter;
+		}
+		return this.parameters;
+	}
+
+	/**
+	 * Deletes an existing parameter.
+	 *
+	 * @param id Id of the parameter to delete.
+	 * @returns The deleted parameter.
+	 */
+	public async deleteParameter(id: number): Promise<ResponseParameter> {
```

Comment:
This call signature is incompatible with the concrete implementation's call signature. The concrete service is `public async deleteParameter(typeOrId: number | ResponseParameter): Promise<void>` while the testing service uses `public async deleteParameter(id: number): Promise<ResponseParameter>`.